bundler/spec/dependabot/bundler/file_parser_spec.rb

```diff
@@ -622,6 +622,20 @@
       end
     end
 
+    context "with a gemspec that loads dependencies from another gemspec dinamically" do
```

Comment:
typo: dinamically -> dynamically

Reply:
Again 😖

Reply:
Corrected in PR title, PR body, commit messages (twice), and specs (twice) 😆